lib/devise_saml_authenticatable/logger.rb

```diff
@@ -1,9 +1,9 @@
 module DeviseSamlAuthenticatable
 
   class Logger    
-    def self.send(message, logger = Rails.logger)
+    def self.send(message, log_level, logger = Rails.logger)
```

Comment:
It might be nice if this defaulted to info:

```suggestion
    def self.send(message, log_level = Logger::INFO, logger = Rails.logger)
```